app/javascript/src/components/Dashboard/Notes/NoteList/DropDownNote.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { Dropdown } from "@bigbinary/neetoui";
+
+import Menu from "./Menu";
+
+const DropDownNote = ({ handleEditClick, handleDeleteClick }) => (
+  <div>
+    <Dropdown customTarget={Menu}>
+      <li onClick={handleEditClick}>Edit</li>
+      <li onClick={handleDeleteClick}>Delete</li>
+    </Dropdown>
+  </div>
```

Comment:
Do we need this `div` here?

Reply:
I will remove the div. It is not required.